packages/actor-scraper/camoufox-scraper/src/internals/crawler_setup.ts

```diff
@@ -2,29 +2,35 @@ import { readFile } from 'node:fs/promises';
 import { dirname } from 'node:path';
 import { fileURLToPath, URL } from 'node:url';
 
-import { browserTools, constants as scraperToolsConstants, CrawlerSetupOptions, createContext, RequestMetadata, tools } from '@apify/scraper-tools';
+import type { AutoscaledPool,
```

Comment:
imports are not always properly formatted, maybe we should really add prettier or biome instead of dealing with those by hand